src/measure_theory/set_integral.lean

```diff
@@ -395,8 +395,8 @@ lemma integral_univ : ∫ x in univ, f x ∂μ = ∫ x, f x ∂μ := by rw [meas
 
 lemma integral_add_compl (hs : measurable_set s) (hfi : integrable f μ) :
   ∫ x in s, f x ∂μ + ∫ x in sᶜ, f x ∂μ = ∫ x, f x ∂μ :=
-by rw [← integral_union disjoint_compl_right hs hs.compl hfi.integrable_on hfi.integrable_on,
-  union_compl_self, integral_univ]
+by rw [← integral_union (@disjoint_compl_right (set α) _ _) hs hs.compl
+    hfi.integrable_on hfi.integrable_on, union_compl_self, integral_univ]
```

Comment:
I'm not sure why the proof here broke; do I need to do something with the `set` instances?